src/markdownExtensions.ts

```diff
@@ -0,0 +1,118 @@
+import { extensions, Disposable, Uri, Extension, Event, EventEmitter } from 'vscode';
```

Comment:
Please add a comment including the link to the VSCode's relevant code just in case we need to have a look in the future.